src/components/story-description/story-description.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { FC, useMemo } from 'react';
+import cx from 'classnames';
+import styles from './story-description.module.scss';
+import { ElementContent } from '../../types';
+import { FlexAlign, FlexDirection, FlexGap, FlexJustify } from 'src/helpers/components/Flex/FlexConstants';
+import Flex from 'src/helpers/components/Flex/Flex';
+
+type StoryDescriptionProps = {
+  align?: FlexAlign;
+  description?: string | ElementContent;
```

Comment:
```suggestion
  description?: ElementContent;
```
Already contains string